homework/homework.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type ID interface {
+	GetId() int
+}
+
+type Metadata interface {
+	GetMetaData() string
+}
+
+func main() {
+	camerasList := buildCamerasList()
+
+	data := map[int]string{}
+	photoID := 1
+	for _, zooCamera := range camerasList {
+		photoData := ""
+		// Check if camera is instance of ID interface
+		if IDCamera, ok := zooCamera.(ID); ok {
+			photoData += fmt.Sprintf("Camera ID: %d, ", IDCamera.GetId())
+		}
+
+		photo := zooCamera.TakePhoto()
+		photoData += photo.Body
+
+		if metadataCamera, ok := zooCamera.(Metadata); ok {
+			photoData += fmt.Sprintf(", Metadata:  %s", metadataCamera.GetMetaData())
+		} else if photoData != "" {
+			photoData += fmt.Sprintf(", Metadata is empty")
+		}
+
+		if photoData != "" {
+			data[photoID] = photoData
+			photoID++
+		}
+	}
+
+	sendDataToServer(data)
+}
+
+var sendDataToServer = func(data map[int]string) {
```

Comment:
Ось так трішки простіше:
```suggestion
func sendDataToServer (data map[int]string) {
```

Reply:
так, але то було зроблено для моків функцій